docs/submitting-applications.md

```diff
@@ -177,8 +177,9 @@ debugging information by running `spark-submit` with the `--verbose` option.
 
 # Advanced Dependency Management
 When using `spark-submit`, the application jar along with any jars included with the `--jars` option
-will be automatically transferred to the cluster. Spark uses the following URL scheme to allow
-different strategies for disseminating jars:
+will be automatically transferred to the cluster. URLs supplied after --jars must be separated by commas. Each entry points to a specific jar file, resulting in a comma-separated list of local jars. That list is included on the driver and executor classpaths. Directory expansion does not work with --jars. 
```

Comment:
It was a dumb question since it's answered just below, yes. I didn't even realize this. They're really URIs not URLs but this is minor.

Actually, `spark-submit --help` refers to local JARs, but yeah they're not necessarily local? I think it'd be fine to also fix the help text on all of the spark-* scripts while we're here.